### DISCUSSION TITLE
Only on Chrome, apparition of html code on top of all pages (!-- document head --> ">) any ideas?

### DISCUSSION THREAD

**Discussion by Chacapamac** [templaza/astroid-framework#252] [Q&A] [answered]
On this website, only in Chrome and in both languages this html code (_!-- document head --> ">_) start to appear on top of all pages????

https://efoilsurf.ca/en/

![chrome-bug](https://user-images.githubusercontent.com/1907276/203719541-05f7ee5d-aea2-49e9-8d2c-dd9f50b06277.jpg)

**W3C Validate give this:**
![validate](https://user-images.githubusercontent.com/1907276/203726016-d8f90be4-5656-43a1-920c-ebf6f0e203ec.jpg)

**In the code:**
![error](https://user-images.githubusercontent.com/1907276/203726913-4736a9fb-f1df-40f5-95da-8940c75ddb13.jpg)

**If I remove the** `<!-- document head -->` comments in the **template Zero > index.php**
**These 2 errors disappears and the (_!-- document head --> ">_) in the head of all page disappears????**

**Comment by sonvnn** [comment, marked as answer]
Hi @Chacapamac,

This is comment code only. You can remove it.

Best Regards,
Sonny

**Comment by Chacapamac** [comment]
Thanks Sonny, this is weird that its causing problem on some copy of that website and the original. 
Anyway its also time to tackle the complete W3C check on that website.

Good Day to all!

**Comment by Chacapamac** [comment]
@sonvnn 
This happen every time I update Astroid

In the **index.html** the comment code is correct
```
<head>
   <astroid:include type="head-meta" /> <!-- document meta -->
   <jdoc:include type="head" /> <!-- joomla head -->
   <astroid:include type="head-styles" /> <!-- head styles -->
   <astroid:include type="head-scripts" /> <!-- head scripts -->
</head> <!-- document head -->
```
In the frontend the code appear with a missing starting bracket (<) and cause the comment to show in the html as text.
Something is wrong with the code before.... I have to remove the comment completely.

**Comment by sonvnn** [comment]
@Chacapamac 
Sorry for the late reply. I just busy for a vacation. May you send me your website again? I will help you check it.

Best Regards,
Sonny

**Reply by Chacapamac** [reply]
**No problem Sonny, it is no rush!**
Anyway, I make a copy of the website with the problem on my testing server at —> http://www.efoilsurf.graf-test.ca/en/

If you look with **Chrome** _(only appear in Chrome)_ you can see that **index.php** code appearing at the top 
—> _!-- document head -->_
and also —> _n>_
These appear on all pages

![Screen Shot 2022-12-07 at 1 09 19 PM](https://user-images.githubusercontent.com/1907276/206262048-61e9284a-2c83-4b48-8448-b3dd921af65e.jpg)

For now I will start to strip as much elements/components/ modules that are common to all pages and verify the HTML VALIDATION again.

If you want I can give you an admin login right now or you can wait when I finish my clean up. 

Good Day to everybody!

**Comment by Chacapamac** [comment]
Ok, I Just discover something, I’m coming back to you soon!

**Comment by Chacapamac** [comment]
Working now, was a weird bug — Solved
